network-api/networkapi/templates/pages/buyersguide/methodology.html

```diff
@@ -59,7 +59,7 @@ <h3 class="p-heading mt-3 mb-1" id="product-selection">
         {% blocktrans with consumer_reports="https://www.consumerreports.org/" wirecutter="https://thewirecutter.com/" cnet="https://cnet.com/" trimmed %}
         When we select products for *Privacy Not Included, our goal is to choose connected products and apps that are likely to
         be popular with consumers in North America and Europe. We make our decisions based on our own research of top selling
-        products that are highly rated across a variety of consumer product websites such as 
+        products that are highly rated across a variety of consumer product websites such as
```

Comment:
The removal of the white space here might be incorrect, as there is no leading space in the href `consumer reports`

Reply:
It seems to be rendering ok: 
<img width="691" alt="image" src="https://user-images.githubusercontent.com/24797493/180877050-9ae06225-1e16-4387-9866-89fa21051d5b.png">